content/id/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html

```diff
@@ -0,0 +1,49 @@
+---
+title: Tutorial Interaktif - Men-<i>Deploy</i> Aplikasi
+weight: 20
+---
+
+<!DOCTYPE html>
+
+<html lang="id">
+
+<body>
+
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+<link href="/docs/tutorials/kubernetes-basics/public/css/overrides.css" rel="stylesheet">
+<script src="https://katacoda.com/embed.js"></script>
+
+<div class="layout" id="top">
+
+    <main class="content katacoda-content">
+
+        <div class="row">
+            <div class="col-md-12">
+                <p>
+                Pod merupakan unit eksekusi utama pada aplikasi Kubernetes. Setiap Pod mewakili sebagian dari beban kerja (<i>workload</i>) yang berjalan pada klaster kamu. <a href="id/docs/concepts/workloads/pods/pod-overview/#understanding-pods">Learn more about Pods</a>.
```

Comment:
Can we translate the hyperlink also?
```suggestion
                Pod merupakan unit eksekusi utama pada aplikasi Kubernetes. Setiap Pod mewakili sebagian dari beban kerja (<i>workload</i>) yang berjalan pada klaster kamu. <a href="/id/docs/concepts/workloads/pods/pod-overview/#understanding-pods">Learn more about Pods</a>.
```